### ISSUE TITLE
a11y: Links should have a desciption

### ISSUE THREAD

**Issue by Richard87** [equinor/radix-web-console#1211]
https://www.visionaustralia.org/business-consulting/digital-access/blog/how-to-make-read-more-links-accessible

We should make sure all links have a `<span class="sr-only">Description</span>` for all links

for example the link to a sites public url looks like this:

![Image](https://github.com/user-attachments/assets/dcb265ce-cb65-429a-a4e1-cd0ed4c88093)

But a screen reader will only read "Link", Adding  `<span class="sr-only">Public Url</span>` to it, would make it read "Link Public Url, Link"
